cms/startup.py

```diff
@@ -21,6 +23,11 @@ def run():
 
     add_mimetypes()
 
+    # Monkey patch the keyword function map
```

Comment:
A future enhancement could be to port this logic over to my fancy `common/djangoapps/monkey_patch` library ;)